modules/sequence/src/test/scala/lucuma/odb/sequence/TestItcClient.scala

```diff
@@ -44,6 +44,9 @@ object TestItcClient {
       ): F[SpectroscopyResult] =
         SpectroscopyResult(Version, result.some).pure[F]
 
+      def optimizedSpectroscopyGraph(input: lucuma.itc.client.OptimizedSpectroscopyGraphInput, useCache: Boolean): F[lucuma.itc.client.OptimizedSpectroscopyGraphResult] =
```

Comment:
This method is needed now but luckily we're not calling it, so I just stubbed it out.